docs/semgrep-assistant/getting-started.md

```diff
@@ -0,0 +1,133 @@
+[<i class="fa-regular fa-file-lines"></i> standard permissions required for Semgrep](/deployment/checklist/#permissions).
+
+Semgrep Assistant requires the **API scope** to run in both GitLab SaaS and GitLab self-managed instances. This can be specified at either the [project access token level](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html) or [personal access token level](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html). 
+
+* You can revoke [project access tokens](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html#revoke-a-project-access-token) or [personal access tokens](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#revoke-a-personal-access-token) at any time. 
+* Semgrep Assistant only accesses source code repositories (projects) on a file-by-file basis; it does not need or request org-level access to your codebase.
+* The token can be configured to limit its scope to specific projects or individuals. You do not need to give read access to all projects in your GitLab organization.
+
+<h2> Enable Assistant </h2>
+
+1. Sign in to [Semgrep Cloud Platform <i class="fas fa-external-link fa-xs"></i>](https://semgrep.dev/login) using your GitLab account.
+2. Click **[<i class="fa-solid fa-gear"></i> Settings](https://semgrep.dev/orgs/-/settings/)**. 
+3. In the **Assistant** section, click the **<i class="fa-solid fa-gear"></i> Allow code snippets in AI prompts** toggle.
+![Screenshot of Semgrep Assistant setup button](/img/semgrep-assistant-setup.png#md-width)
+   This launches the **Set up Semgrep Assistant** prompt.
+4. Follow the on-screen instructions to complete the setup process.
+5. Navigate back to the **Deployment** page. Under the **Assistant** section, verify that all of the features are enabled:
+   1. **Allow code snippets in AI prompts**: Required for Semgrep to auto-triage findings, provide AI remediation guidance, and tag findings with code context.
+   2. **Autofix suggestions for Code**: Enable autofix suggestions in comments from Assistant. You can also set the minimum confidence level for Assistant-written fixes if the Semgrep rule doesn't include a human-written autofix.
+   3. **Auto-triage for Code**: Enable notifications whenever Assistant suggests that a finding may be safe to ignore. You can include notifications in your PR and MR comments, or you can receive them through Slack notifications.
+    ![Screenshot of Semgrep Assistant toggle location](/img/semgrep-assistant-enable.png)
+
+</TabItem>
+</Tabs>
+
+### Enable autofix suggestions
+
+If [autofix](/semgrep-assistant/overview/#autofix), which allows you to receive code snippets to remediate true positive findings, isn't enabled for your deployment, you can do so as follows:
+
+1. Log in to Semgrep Cloud Platform, and navigate to **Settings > Deployment**.
```

Comment:
```suggestion
1. Sign in to Semgrep Cloud Platform, and navigate to **Settings > Deployment**.
```

Have you ever wondered why some of our procedures are "log in" vs "sign in"? It's to match the app's UI text, which says "Sign in through...", whereas the CLI says `login` 😅